server/server.js

```diff
@@ -41,4 +41,15 @@ app.post('/api/hostels/:hostel_id/images', (req, res) => {
   });
 });
 
+app.delete('/api/hostels/:hostel_id/images', (req, res) => {
+  console.log(req.body);
+  models.image.deleteEntry(req.body.id, (error, deletedImage) => {
+    if (error) {
+      res.status(502).send();
+      return;
+    }
+    res.send(deletedImage);
```

Comment:
You can probably just say the image was deleted instead of sending back the image data.